asset/cpu_triggered_asset_utilization.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+import argparse
+import getpass
+import sys
+import time
+import winreg
+
+with winreg.OpenKey(
```

Comment:
This logic should be put into a private function but called globally (since you need the path for an import).